D3DBase.py

```diff
@@ -0,0 +1,6 @@
+import os
+
+__dir__ = os.path.dirname(__file__)
+ICON_PATH = os.path.join( __dir__, 'Resources/icons' )
+IMAGE_PATH = os.path.join( __dir__, 'Resources/images' )
+TABLE_PATH = os.path.join( __dir__, 'tables' )
```

Comment:
Should this go in Resources too?

Reply:
You are right. But there is some difference between the table-resource and icons/image-resource.
The tables supposed to be editited but the other things not.